adbe/version.txt

```diff
@@ -1 +1 @@
-2.5.4
\ No newline at end of file
+2.5.5
```

Comment:
Please revert the change to `version.txt`. This file contains the version which is only incremented at the time of the release.